trailblazer/store/crud/delete.py

```diff
@@ -10,3 +17,18 @@ def delete_analysis_jobs(self, analysis: Analysis) -> None:
         for job in analysis.jobs:
             job.delete()
         self.commit()
+
+    def delete_analysis(self, analysis_id: int, force: bool = False) -> None:
+        """Delete the analysis from the database. Also delete ongoing analysis if 'force' is True.
+        Raise: MissingAnalysis when no analysis and TrailblazerError for ongoing analysis for analysis id.
```

Comment:
Would you consider formatting the error-raising description in docstrings like this? With the description of the exception in another line and preceded with a tab and the different exception types on different lines. We have not discussed this, so feel free to disregard it. If you like the suggestion, implement it in the text editor and not here, as GitHub does not allow me to add tabs, I just added some spaces
```suggestion
        """Delete the analysis from the database. Also delete ongoing analysis if 'force' is True.
        Raises: 
            MissingAnalysis when no analysis is found 
            TrailblazerError when the analysis is ongoing.
```